src/App/init.sql

```diff
@@ -0,0 +1,262 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 15.9 (Debian 15.9-1.pgdg120+1)
+-- Dumped by pg_dump version 15.9 (Debian 15.9-1.pgdg120+1)
+
+SET statement_timeout = 0;
```

Comment:
why not just use the defaults? If you are not correct in your configuration, it can throw off a lot. if you dont know what it does remove it.